seedlet/assets/js/editor-dark-support.js

```diff
@@ -0,0 +1,66 @@
+( function( domReady ) {
+
+	/* 
+		TODO: 
+		  Determine when all the CSS variables have been loaded.
+			Is that something we can add as a dependency to the  wp_enqueue_script?
+			If so I haven't yet been able to get that to work.
+
+		  Determine when the editor styles wrapper has been loaded.
+			Alternative tester element included below
+
+		  Replace brightnessByColor function (random copy/pasta) with standard color luminosity utils (is there one?)
+		  Ensure Lumenosity value is approprate for switching
+	*/
+	    
+    // domReady( function() {
+	setTimeout(()=>{
```

Comment:
Is there something we could listen to/wait for rather than just waiting 3 seconds?